### DISCUSSION TITLE
Question: Multiple tabs within the ControNet window for different Control Model

### DISCUSSION THREAD

**Discussion by consecutiotemporum** [Mikubill/sd-webui-controlnet#574] [Q&A] [answered]
How do I get multiple tabs within the ControNet window for different Control Model?

![Screenshot 2023-03-12 193307](https://user-images.githubusercontent.com/114416380/224565208-84418605-695b-405f-bd34-0b3783a9bccc.png)

**Comment by ljleb** [comment, marked as answer]
raise the value of "Multi ControlNet: Max models amount (requires restart)" under Settings tab > ControlNet

**Reply by consecutiotemporum** [reply]
Thx!

**Comment by concaption** [comment]
What does multiple tabs signify? is one tab going into another input or something else??
